README.md

```diff
@@ -80,66 +103,115 @@ Current adjacent projects that are within the TSC's scope of responsibility can
-* Creating new repositories and projects under the _nodejs_ GitHub organization as required
-* Setting overall technical direction for the Node.js Core project, including high-level goals and low-level specifics regarding features and functionality
-* Setting and maintaining appropriate standards for community discourse via the various mediums under TSC control
-* Setting and maintaining governance rules for the conduct and make-up of the TSC, CTC, Working Groups and other bodies within the TSC's domain
-
-Many of these responsibilities will be delegated by the TSC to appropriate bodies such as the CTC and Working Groups.
+* Managing code and documentation creation and changes for the listed projects
+  and resources
+* Setting and maintaining standards covering contributions of code,
+  documentation and other materials
+* Managing code and binary releases: types, schedules, frequency, delivery
+  mechanisms
+* Making decisions regarding dependencies of the Node.js Core project,
+  including what those dependencies are and how they are bundled with source
+  code and releases
+* Creating new repositories and projects under the _nodejs_ GitHub organization
+  as required
+* Setting overall technical direction for the Node.js Core project, including
+  high-level goals and low-level specifics regarding features and functionality
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+
+Many of these responsibilities will be delegated by the TSC to appropriate
+bodies such as the Working Groups.
 
 ## TSC Members
 
-TSC members are responsible for top level technical community concerns. The role is
-mostly administrative and is responsible for admitting new Top Level Projects, Top Level
-Working Groups, and advocating for any needs in the technical side of the foundation to
-the Node.js Foundation Board of Directors.
-
-TSC members can nominate new members at any time. Candidates for membership tend to be people
-who have a competancy for community management and a high tolerance and patience for process
-minutiae as the TSC delegates most of its responsibilities to other projects and working groups.
+TSC members are responsible for top level technical community concerns. The role
+is mostly administrative and is responsible for admitting new Top Level
+Projects, Top Level Working Groups, and advocating for any needs in the
+technical side of the foundation to the Node.js Foundation Board of Directors.
 
-Every Top Level Project not currently incubating can appoint someone to the TSC who they elect
-at their own discretion.
+Every Top Level Project not currently incubating can appoint someone to the TSC
+who they elect at their own discretion.
```

Comment:
Worst nit ever: `who` -> `whom` (I'm not actually a stickler on the `whom` usage stuff, but might as well preempt a grammar discussion by just using `whom`. People may disagree about whether `who` is OK here but no one should disagree about `whom` in this situation.)